### DISCUSSION TITLE
IVF_PQ memory usage calculation result in sizing-tool has changed.

### DISCUSSION THREAD

**Discussion by rky0930** [milvus-io/milvus#4896] [Q&A and General discussion]
Suddenly, at https://www.zilliz.com/sizing-tool, 
the result of calculating IVF_PQ's memory usage increased. In the current state, IVF_PQ's memory capacity is too large to be used. Is the current value normal?

The result of calculating with the same option yesterday(2021.04.01) was about 40Gb.

<img width="1273" alt="스크린샷 2021-04-02 오후 6 39 09" src="https://user-images.githubusercontent.com/5013450/113404202-bf852c80-93e2-11eb-931a-02b993110956.png">

**Comment by nameczz** [comment]
We update sizing-tool yesterday.  Fix https://github.com/milvus-io/milvus/issues/4886.

But we are not change the IVF_PQ memory algorithm, also we thought 191GB is correct in your case.

**Reply by rky0930** [reply]
@nameczz Thanks for replying !
If so, have there been any changes in IVF_PQ memory calculation method within a month? If the sizing-tool is opensource, May I check how the memory usage is calculated? ?

Since commit of the change is not linked to #4886, I cannot see how it has changed.
